content/sql/concepts/commands/terms/contains/contains.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'CONTAINS'
```

Comment:
After taking a closer look, let's actually move this to a term under ["operators"](https://www.codecademy.com/resources/docs/sql/operators) rather than commands. Thanks in advance for moving this!

The reason is that SQL operators are used in the `WHERE` clause to perform some type of operation.